tests/unit/transformer/test_mapper.py

```diff
@@ -192,6 +192,58 @@ def test_empty_input_dataframe_raises_exception(self, spark_session, transformer
             transformer.transform(input_df)
 
 
+class TestNullifyMissingColumns(object):
+    """
+    Nullify input columns in case it does not exist
+    """
+
+    @pytest.fixture(scope="class")
+    def transformer(self, mapping):
+        return Mapper(mapping=mapping, skip_missing_columns=False, nullify_missing_columns=True)
+
+    @pytest.fixture(scope="class")
+    def mapped_df(self, input_df, transformer):
+        input_df = input_df.drop("attributes")
+        return transformer.transform(input_df)
+
+    def test_missing_columns_are_not_skipped(self, mapped_df, mapping):
+        assert len(mapping) == len(mapped_df.columns)
+
+    def test_missing_columns_are_nullified(self, mapped_df, mapping):
+        attribute_columns = [item[0] for item in mapping if item[1].startswith("attributes.")]
```

Comment:
Makes it a bit easier to read:
```suggestion
        attribute_columns = [name for name, source, _ in mapping if source.startswith("attributes.")]
```
Same goes for L233